code/lib/cli/src/upgrade.ts

```diff
@@ -37,6 +37,18 @@ export const getStorybookVersion = (line: string) => {
   };
 };
 
+const getInstalledStorybookVersion = async (packageManager: JsPackageManager) => {
+  const installations = await packageManager.findInstallations(['storybook', '@storybook/cli']);
+  if (!installations) {
+    return;
+  }
+  const cliVersion = installations.dependencies['@storybook/cli']?.[0].version;
```

Comment:
`installations.dependencies['@storybook/cli']?.[0]` -> Can this be undefined? If so `installations.dependencies['@storybook/cli']?.[0].version` will through.

Reply:
It can't, TS would complain, and the current code producing it doesn't do that either.